content/docs/custom-commands/commands.md

```diff
@@ -242,6 +252,61 @@ The modal trigger is used to trigger custom commands via submitting a modal.
 
 The trigger is matched using [RegEx](/docs/reference/regex).
 
+##### Crontab
+
+This trigger will run the command at scheduled intervals using [Cron Job Scheduling](https://en.wikipedia.org/wiki/Cron). This allows for advanced specificity in scheduling execution, such as scheduling execution at 23:45 every Saturday.
+
+When using an interval trigger, the custom command does not receive any user or member context. Thus, `{{ .User.ID }}`
```

Comment:
when using a cron trigger

Reply:
I wonder if we could instead change the term to "time trigger" to maintain the umbrella statement encompassing both interval and cron without adding more confusion between cron and interval. Thoughts?